### DISCUSSION TITLE
Novice here! trying bypass power checking - "TUSB=1" not working - what am i doing wrong?

### DISCUSSION THREAD

**Discussion by Tori456** [gopro/labs#734] [Q&A]
Hi! im trying to bypass power checking on my go pro so i can use just a power bank to power it. I saw that people say to use command "TUSB=1" and it should work but it's not working for me? I have been typing it in to the bit at the bottom of the qr creator page is that correct?
What am i doing wrong? any advice is super appreciated thank you:) 
<img width="814" alt="Screenshot 2024-01-24 at 13 21 59" src="https://github.com/gopro/labs/assets/157606731/c3924232-1423-4fd5-b9c0-b5c7619e81de">

**Comment by dnewman-gpsw** [comment]
The command is !MTUSB=1

![image](https://github.com/gopro/labs/assets/10856084/110403ad-f701-46c8-b418-2cabbdb82ed6)

**Reply by Tori456** [reply]
This worked perfectly thank you so much :)
